cmd/sdk-server/main.go

```diff
@@ -164,39 +170,57 @@ func main() {
 	logger.Info("shutting down sdk server")
 }
 
-func registerLocal(grpcServer *grpc.Server, ctlConf config) (localSDK *sdkserver.LocalSDKServer, err error) {
+// registerLocal registers the local SDK servers, and returns a cancel func that
+// closes all the SDK implementations
+func registerLocal(grpcServer *grpc.Server, ctlConf config) (func(), error) {
 	filePath := ""
 	if ctlConf.LocalFile != "" {
-		filePath, err = filepath.Abs(ctlConf.LocalFile)
+		filePath, err := filepath.Abs(ctlConf.LocalFile)
 		if err != nil {
-			return
+			return nil, err
 		}
 
 		if _, err = os.Stat(filePath); os.IsNotExist(err) {
-			err = errors.Errorf("Could not find file: %s", filePath)
-			return
+			return nil, errors.Errorf("Could not find file: %s", filePath)
 		}
 	}
 
-	localSDK, err = sdkserver.NewLocalSDKServer(filePath)
+	stableSDK, err := sdkserver.NewLocalSDKServer(filePath)
 	if err != nil {
-		return
+		return nil, err
 	}
-	sdk.RegisterSDKServer(grpcServer, localSDK)
-	return
+	alphaSDK := serveralpha.NewLocalSDKServer()
+	betaSDK := serverbeta.NewLocalSDKServer()
+
+	sdk.RegisterSDKServer(grpcServer, stableSDK)
+	sdkalpha.RegisterSDKServer(grpcServer, alphaSDK)
+	sdkbeta.RegisterSDKServer(grpcServer, betaSDK)
+	return func() {
+		stableSDK.Close()
```

Comment:
Should we also close the alpha and beta sdks?  The function comment makes it sound like all SDKs will be cleaned up.

Reply:
That's an interesting question. Neither the alpha or beta SDKs have/need a Close() operation. 

Should we add empty ones, just for consistency?

Reply:
Since alpha / beta are currently just empty stubs, I can't tell if they will grow to need a close operation (in which case we should add it now so we don't forget to add it here later) or whether we don't expect them to ever need a close operation. In the latter case, I think it's fine to update the function comments. Something like

"... SDK servers, and returns a function that cleans up the SDK implementations when they are finished being used."

Maybe that isn't any better than what you had. :)

Reply:
Since I don't think we can guarantee it either way - I've added a stub function in both alpha and beta SDK, and called it in both returned functions.

At least then if someone _does_ need it, it's easy to see where to place tear down logic.